zip-####.rst

```diff
@@ -155,62 +207,77 @@ SHOULD be applied to the Zcash Testnet for testing purposes.
 Dev Fund allocation
 -------------------
 
-Starting at the first Zcash halving in 2020, until the second halving in 2024,
-20% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
-consists of the following three slices:
+Starting at the second Zcash halving in 2024, until the third halving in 2028,
+15% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
+consists of the following allocations:
 
-* 35% for the Bootstrap Project (denoted **BP slice**);
-* 25% for the Zcash Foundation, for general use (denoted **ZF slice**);
-* 40% for additional "Major Grants" for large-scale long-term projects
-  (denoted **MG slice**).
+* 5% for the Zcash Foundation (for internal work and grants);
+* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
+and contingent upon an ecosystem mandated "non-direct funding model" for the future;
```

Comment:
Given that the ZIP mandates the ecosystem to design and deploy a NDFM, the "contingent upon" appears superfluous:
```suggestion
* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only;
```